.github/CODEOWNERS

```
@@ -1,2 +1,21 @@
 # global code owner:
-*       @davidmurdoch
+
+-       @davidmurdoch
```

was this line supposed to change?

---

```suggestion
*       @davidmurdoch
```
Good find don't know how that happened!